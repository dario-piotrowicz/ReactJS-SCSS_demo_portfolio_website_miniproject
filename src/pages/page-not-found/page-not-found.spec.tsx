import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './page-not-found.page.component';

describe('page-not-found page', () => {
  it('should match its snapshot', () => {
    expect(shallow(<PageNotFound />)).toMatchSnapshot();
  });
});
