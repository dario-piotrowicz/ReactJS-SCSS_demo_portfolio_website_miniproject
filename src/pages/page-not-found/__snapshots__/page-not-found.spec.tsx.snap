// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page-not-found page should match its snapshot 1`] = `
<div
  id="page-not-found"
>
  <h1>
    404 - Page Not Found!
  </h1>
</div>
`;
